### ISSUE TITLE
[Feature Request] Bloom filter not Open Source 

### ISSUE THREAD

**Issue by raphaelauv** [delta-io/delta#1347]
## Feature request

Look like in 2.0 and 2.1rc1 the Bloom filters are not opensource

any plan to release them in the open source version of delta ?

Thanks

**Comment by dennyglee** [comment]
Yes, I think we will want to implement bloom filters, but we may need to consider a slightly different design.  [Apache Parquet currently supports bloom filters](https://github.com/apache/parquet-format/blob/master/BloomFilter.md), and we may want to leverage this.

**Comment by dudzicp** [comment]
Hi @dennyglee is there an update on this?

**Comment by dennyglee** [comment]
We're still working on publishing the roadmap but we're a tad behind schedule.  Will get to this shortly - thanks for your patience!

**Comment by ugurkalkavan** [comment]
Hi @dennyglee is there any update on this?

**Comment by swpnk** [comment]
Hey any update on this? @dennyglee

**Comment by foss-contributor** [comment]
@raphaelauv, thank you for your contribution! This issue has been inactive for over 18 months and will be closed in one week. To keep it open, please leave a comment on this issue or contact us in the `#contributors` channel on [Delta Lake Slack](https://go.delta.io/slack). We appreciate your involvement in Delta Lake!

**Comment by raphaelauv** [comment]
No stale
